pkg/postage/postagecontract/contract_test.go

```diff
@@ -28,10 +28,8 @@ import (
 var postageStampContractABI = abiutil.MustParseABI(chaincfg.Testnet.PostageStampABI)
 
 func TestCreateBatch(t *testing.T) {
-	defer func(b uint8) {
```

Comment:
By making this test run in paralel, other tests that use `postagecontract.BucketDepth` may have read an incorrect value (in this case 9) or a data race if they also want to set value of that global variable.

As this test changes `postagecontract.BucketDepth` it should not run in paralel. Also, it should put back the original value when it finishes, maybe even better to use t.Cleanup than defer and make subtests here parallel.